Sources/Controllers/Base/OABaseButtonsViewController.m

```diff
@@ -67,9 +70,17 @@ - (void)updateBottomButtons
 
     UIColor *bottomBackgroundColor = [self getBottomBackgroundColor];
     if (bottomBackgroundColor)
+    {
         self.bottomBackgroundView.backgroundColor = bottomBackgroundColor;
+    }
     else
-        [self.bottomBackgroundView addBlurEffect:YES cornerRadius:0. padding:0.];
+    {
+        if (!_isBlurEffect)
+        {
+            [self.bottomBackgroundView addBlurEffect:YES cornerRadius:0. padding:0.];
```

Comment:
It is ok to add few blur effects to view? If not it should be implemented at addBlurEffect as well.
Should `light` parameter be theme dependent?